openquake/mbt/tests/tools/fault_modeler/data/fault_database.base.geojson

```diff
@@ -0,0 +1 @@
 [93.31166278416347, 49.74838934452688], [93.31402201071367, 49.749088024416544], [93.31628293615759, 49.74978669424257], [93.31913366823909, 49.750612381971536], [93.32237760474561, 49.75181913106104], [93.32630964899596, 49.75251776154784], [93.32886547775867, 49.7538514827143], [93.33201111315894, 49.7549311348905], [93.33358393085906, 49.75531218286116], [93.33545165187797, 49.75543919818605], [93.33702446957811, 49.75562972054962], [93.33908879280953, 49.75632829614549], [93.33977690055333, 49.756582321139895], [93.34174292267849, 49.756582321139895], [93.3429225359536, 49.75670933313809], [93.3452817625038, 49.75639180251884], [93.34823079569156, 49.75632829614549], [93.3516713344106, 49.75645530880903], [93.35363735653576, 49.75734438813908], [93.35604573363908, 49.75880498307974], [93.3581100568705, 49.75956701514917], [93.36017438010191, 49.76061478969038], [93.3619929505677, 49.761503792757516], [93.36371321992722, 49.762773768863255], [93.36725205975252, 49.76391671891639], [93.36985703906836, 49.765948563591465], [93.37064344791844, 49.76747239121267], [93.37285522280925, 49.76851999491588], [93.37575510544386, 49.77013897428815], [93.37840923531284, 49.77125000727418], [93.38125996739433, 49.772805410668994], [93.38558521606969, 49.77499558795741], [93.38809189427928, 49.776169990082984], [93.38991046474504, 49.77683652998003], [93.39340015401721, 49.77782045307194], [93.39556277835491, 49.77915347829448], [93.39983887647715, 49.78035951713472], [93.40495053400255, 49.78175594565727], [93.40681825502148, 49.782358936440886], [93.40996389042176, 49.78442174289538], [93.41192991254692, 49.78543724611714], [93.41463319296902, 49.78642099456357], [93.41615686011603, 49.7880076435104], [93.41920419441004, 49.78937212003929], [93.42239898036343, 49.790006747189025], [93.42564291686995, 49.790355788577074], [93.4285919500577, 49.790673096747156], [93.43163928435169, 49.790800019433135], [93.43439171532691, 49.79127597654335], [93.43768480238657, 49.79200576836154], [93.44019148059616, 49.792640361004004], [93.44338626654955, 49.793560505566205], [93.4451556864622, 49.794131620984246], [93.44967753735008, 49.79565456250559], [93.45252826943158, 49.797114003188845], [93.45454344210985, 49.798097514492454], [93.45788567972265, 49.79889065421891], [93.4653565637983, 49.80123827166834], [93.4743019644678, 49.80415677247644], [93.4803966330558, 49.807519174845766], [93.48767091491892, 49.81075447361437], [93.49179956138175, 49.812657489521314], [93.49750102554474, 49.815575301917924], [93.50310418860147, 49.81754155452508], [93.50919885718947, 49.819127183915555], [93.51843916117775, 49.822742224637324], [93.52473043197828, 49.825088685446985], [93.53288942379771, 49.82895692640536], [93.53701807026056, 49.83314188789145], [93.55274624726188, 49.840242812088356], [93.56100354018758, 49.842778603739724], [93.56178994903765, 49.843032175589876]]}, "type": "Feature"}], "crs": {"properties": {"name": "urn:ogc:def:crs:OGC:1.3:CRS84"}, "type": "name"}}
```

Comment:
Is there a specific reason why you don't use indented version of this geojson file ?

Reply:
Same problem also when exporting with python. The formatting has to be done manually later. I agree with @cossatot that it is not worth to spend time in doing this.